internal/action/command.go

```diff
@@ -344,14 +349,40 @@ func ReloadConfig() {
 	if err != nil {
 		screen.TermMessage(err)
 	}
+
+	if reloadAll {
+		err = config.RunPluginFn("deinit")
```

Comment:
This probably should be done *before* reinitializing the list of plugins in `InitRuntimeFiles()`.